lighthouse-core/gather/gather-runner.js

```diff
@@ -187,7 +190,14 @@ class GatherRunner {
       .concat(passContext.settings.blockedUrlPatterns || []);
     const blankPage = passContext.passConfig.blankPage;
     const blankDuration = passContext.passConfig.blankDuration;
-    await GatherRunner.loadBlank(passContext.driver, blankPage, blankDuration);
+
+    // On the very first pass we're already on blank
+    const skipLoadBlank = passContext.firstPass;
+    const pass = skipLoadBlank
+      ? Promise.resolve()
+      : GatherRunner.loadBlank(passContext.driver, blankPage, blankDuration);
+    await pass;
```

Comment:
WDYT about something like

```js
if (!passContext.isFirstPass) {
  // If we're not on the first pass, we're still on the target page.
  // Navigate to blank for the pass setup.
  await GatherRunner.loadBlank(passContext.driver, blankPage, blankDuration);
}
```

the ternary no-op await is just throwin' me for a loop :)

Reply:
better! I refactored the old code (using this structure), and it was using Promise.then, so that's why this made sense at one point :P